### DISCUSSION TITLE
React hook form MUI V5 Autocomplete Yup schema validation not working

### DISCUSSION THREAD

**Discussion by tawhidjony** [react-hook-form/react-hook-form#9885] [General]
[screencast-bpconcjcammlapcogcnnelfmaeghhagj-2023.02.07-17_55_02.webm](https://user-images.githubusercontent.com/20874292/217238551-bb981940-3826-40fe-9bae-0c216ce653a5.webm)
![image](https://user-images.githubusercontent.com/20874292/217238748-8e0b8f9c-8c2d-4f33-b8cb-a127ed211e1f.png)
![image](https://user-images.githubusercontent.com/20874292/217238833-ac458d3f-c22e-4459-9a18-fcaab58002b1.png)

someone help me how can i solved it

**Comment by leapful** [comment]
@tawhidjony 

Could you please provide a codesandbox url to re-produce the issue ?
